gcloud-java-contrib/gcloud-java-nio/pom.xml

```diff
@@ -41,18 +41,11 @@
       <groupId>com.google.auto.service</groupId>
       <artifactId>auto-service</artifactId>
       <version>1.0-rc2</version>
-      <scope>provided</scope> <!-- to leave out of the all-deps jar -->
     </dependency>
     <dependency>
       <groupId>com.google.auto.value</groupId>
       <artifactId>auto-value</artifactId>
       <version>1.1</version>
```

Comment:
According to the public [docs](https://github.com/google/auto) `provided` is correct  for auto-value but not for auto-service (which suggests default/compile with optional settings). I looked at the code and I think that also auto-service's scope should be defined as `provided` (as it was before - my apologies).  I verified it with the guava team and they agreed with me (I filed this [issue](https://github.com/google/auto/issues/305). So I suggest to put back the `provided` scope to both auto-service and auto-value.